tuftool/src/error.rs

```diff
@@ -163,8 +176,11 @@ pub(crate) enum Error {
     },
 
     #[snafu(display("Unable to initialize logger: {}", source))]
-    Logger {
-        source: simplelog::TermLogError,
+    Logger { source: simplelog::TermLogError },
+
+    #[snafu(display("Unable to load incoming metadata"))]
```

Comment:
Display the source error for this and all errors that have a `source`.

```suggestion
    #[snafu(display("Unable to load incoming metadata: {}", source))]
```